### PR TITLE
LOG-5524: use get(object, path) function to get the value of a given path

### PR DESCRIPTION
**Pull request by vparfonov**
### Description
Use` get(object, path)` function to get the value of a given path: e.g.: for label `kubernetes.labels.my-typekey` will use `val = get!(value: ., path: ["kubernetes","labels","my-typekey"])`

/cc @cahartma @Clee2691 
/assign @jcantrill 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA: https://issues.redhat.com/browse/LOG-5524
- Enhancement proposal:

### CODE REVIEW

File: internal/generator/vector/output/elasticsearch/elasticsearch.go [hunk 1]
```diff
@@ -248,3 +249,15 @@ func BasicAuth(id string, o logging.OutputSpec, secret *corev1.Secret) []Element
 
 	return conf
 }
+
+func safePath(path string) string {
+	if !strings.Contains(path, "-") {
```
Comment (jcantrill) [inline internal/generator/vector/output/elasticsearch/elasticsearch.go:254]: We can just make it always use "get" .  It should work for both cases

Comment (vparfonov) [reply internal/generator/vector/output/elasticsearch/elasticsearch.go:254]: yes., it will work for sure

**Pull request merged**